lib/sparql.jison

```diff
@@ -771,23 +771,23 @@ PathProperties
     ;
 
 PathProperty
-    : 'START' Var '='? PathValue 
+    : 'START' Var '='? PathValue?
     {
       if ($4) {
         $$ = { property: 'start', value: { var: $2, value: $4 } };
       } else {
         $$ = { property: 'start', value: { var: $2, value: null } };
       }
     }
-    | 'END'  Var '='? PathValue 
+    | 'END'  Var '='? PathValue?
     {
       if ($4) {
         $$ = { property: 'end', value: { var: $2, value: $4 } };
       } else {
         $$ = { property: 'end', value: { var: $2, value: null } };
       }
     }
-    | 'VIA' Var '='? PathViaValue 
+    | 'VIA' Var '='? PathViaValue
```

Comment:
There should be no equal sign. `VIA` requires a `PathViaValue`, which should be defined as one of Var, PropertyPath, or Pattern.

Reply:
It got overlooked, but now I removed it.